### PR TITLE
Blog post on new NAS report

### PR DESCRIPTION
**Pull request by ehsteve**

### CODE REVIEW

File: _posts/2019-05-17-nas-Reproduce-and-Replicate-in-Science.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "New Report Examines Reproducibility and Replicability in Science, Recommends Ways to Improve Transparency and Rigor in Research."
+author: ehsteve
+---
+
+A new report by the National Academy of Sciences has issued new recommendations to improve confidence in science
```
Comment (namurphy) [inline _posts/2019-05-17-nas-Reproduce-and-Replicate-in-Science.markdown]: I'd suggest removing the second `new`

File: _posts/2019-05-17-nas-Reproduce-and-Replicate-in-Science.markdown [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+through enhancing Reproducibility and Replicability. Often used interchangeably, these two terms are defined
+precisely in the report. Reproducibility means obtaining consistent results through computation with the the same 
+input data while replicability means obtaining consistent results across experiments with their own data.
+
+The PyHC organization is directly aligned with their recommendation to "investing in research and development of open-source,
```
Comment (namurphy) [inline _posts/2019-05-17-nas-Reproduce-and-Replicate-in-Science.markdown]: Perhaps chance `recommendation to` to `recommendation of`

File: _posts/2019-05-17-nas-Reproduce-and-Replicate-in-Science.markdown [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+The PyHC organization is directly aligned with their recommendation to "investing in research and development of open-source, 
+usable tools and infrastructure that support reproducibility for a broad range of studies across 
+different domains in a seamless fashion."
+
+The study was produced by the Committee on Reproducibility and Replicability in Science and funded by
+the National Science Foundation and Alfred P. Sloan Foundation.
+
+A [short summary](http://www8.nationalacademies.org/onpinews/newsitem.aspx?RecordID=25303) is available or 
+get the free [full report](https://www.nap.edu/catalog/25303/reproducibility-and-replicability-in-science).
```
Comment (namurphy) [inline _posts/2019-05-17-nas-Reproduce-and-Replicate-in-Science.markdown]: Maybe something like:
```
A [short summary](http://www8.nationalacademies.org/onpinews/newsitem.aspx?RecordID=25303) and 
the [full report](https://www.nap.edu/catalog/25303/reproducibility-and-replicability-in-science) have been made openly available.
```

File: _posts/2019-05-17-nas-Reproduce-and-Replicate-in-Science.markdown [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+layout: post
+title: "New Report Examines Reproducibility and Replicability in Science, Recommends Ways to Improve Transparency and Rigor in Research."
+author: ehsteve
+---
+
+A new report by the National Academy of Sciences has issued new recommendations to improve confidence in science
+through enhancing Reproducibility and Replicability. Often used interchangeably, these two terms are defined
+precisely in the report. Reproducibility means obtaining consistent results through computation with the the same 
+input data while replicability means obtaining consistent results across experiments with their own data.
+
```
Comment (mbobra) [inline _posts/2019-05-17-nas-Reproduce-and-Replicate-in-Science.markdown]: I would also reference the [NAS report on Open Source Software Policy Options for Earth and Space Sciences](https://www.nap.edu/catalog/25217/open-source-software-policy-options-for-nasa-earth-and-space-sciences). It recommends the following: 

> NASA Science Mission Directorate should explicitly recognize the scientific value of open source software and incentivize its development and support, with the goal that open source science software becomes routine scientific practice.

**Comment by ehsteve** [comment]
Thanks for the feedback @namurphy @mbobra!

**Pull request merged**